ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -82,22 +80,20 @@ export default function OnboardingMetametrics() {
         },
       );
 
-      if (participateInMetaMetrics) {
```

Comment:
it looks like this PR removes the check ensure the user is already participating in meta metrics, right?

Reply:
This state is set to true on the condition above, hence this if condition didn’t make sense. It was routing the user to next page without setting the track event

Reply:
understood, approved